### ISSUE TITLE
Pivot A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7434]
Tracking all of the issues brought up with the 12/18 accessibility pass on Pivot control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B07b9e428-09cf-4a02-8fa1-05015209994a%7D&action=default&uid=%7B07B9E428-09CF-4A02-8FA1-05015209994A%7D&ListItemId=1635&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [ ] Bug1: [Accessibility][Programmatic access]: ARIA roles used must conform to valid values. 
 - [ ] Bug2: [Accessibility][Programmatic access]: 'Browser button' must have discernible text. 
 - [ ] Bug3: [Accessibility][Keyboard navigation]: Keyboard focus indicator is not properly visible on 'Bar' button in forward navigation. 
 - [ ] Bug4: [Accessibility][Programmatic access]: User does not get state change feedback after selecting 'Hide first pivot item' button.  
 - [ ] Bug5: [Accessibility][Keyboard navigation]: Focus order is not correct after 'Go to Microsoft' link. Next keyboard focus goes to the top of the browser control i.e. see all your tabs.  
 - [ ] Bug6: [Accessibility][Screen Reader]: 'Call out' pop up is not accessible in narrator scan mode.  
 - [ ] Bug7: [Accessibility][Usable]: After selecting 'Spinner' tab item even if user moves to the next control narrator keeps reading about 'Spinner alerts' .  
 - [ ] Bug8: [Accessibility][Usable]: Narrator focus is not in sync with keyboard focus while selecting 'Hide/Show first pivot item' button.  
 - [ ] Bug9: [Accessibility][Keyboard navigation]: 'Recent' tab item has two keyboard focuses.  
 - [ ] Bug10: [Accessibility][Usable]: 'My files' tab item should have one scan mode focus and all information should be conveyed on same focus.  
  
  
Component: Pivot 
@micahgodbolt

**Comment by tpanpalia** [comment]
Unable to navigate to succeeding pivot item through tab.
We are able to navigate to pivot items using arrow keys.Is there any plan to make changes so that we can navigate using tab.

**Comment by micahgodbolt** [comment]
#1 the labeled by and ID match, and the role of the button is tab. I don't see the problem 

![image](https://user-images.githubusercontent.com/1434956/50619945-68b1e500-0eb1-11e9-9a82-6936fb1764d5.png)

**Comment by TestAccessibilityGIT** [comment]
Bug 1: We got this defect using Keros for Web tool. Role used is not widely supported in assisstive technologies in this case.

**Comment by TestAccessibilityGIT** [comment]
Hi Micah,
I guess Fernanda Bonnin ferbonn@microsoft.com<mailto:ferbonn@microsoft.com> from Keros can help us on this.

Regards,
Arvind

**Comment by PerthCharern** [comment]
> Unable to navigate to succeeding pivot item through tab.
> We are able to navigate to pivot items using arrow keys.Is there any plan to make changes so that we can navigate using tab.

We are running to this as well. Also, when we have a Pivot inside a Dialog, we can't even tab to the first tab. @micahgodbolt Any plans to change the tabbing behavior above?

**Comment by micahgodbolt** [comment]
Pivots are part of a focus zone, they (like other types of navigation) are accessed through arrow keys once tabbing lands you inside of the pivot control. We don't have any plan to change that behavior, as we don't consider it a bug.

As for pivots inside of Dialogs, do you have a codepen repro of that issue?

**Comment by PerthCharern** [comment]
@micahgodbolt I guess that's fair. The tabbing flow is just a little awkward when you have something on the right of the pivot, and you instinctively think that you can get there by tabbing. It would be nice to offer tabbing as an alternative behavior (as a property flag maybe). In a two-dimensional focus zone, an arrow key makes perfect sense - but when there's only one dimension, tab works much more naturally.

I am trying to repro the problem on CodePen with minimal example - it seems to be a bit more complicated than just a Dialog and a Pivot. Will get back to you once I have singled out the issue.

**Comment by betrue-final-final** [comment]
It's less about dimension and more about establishing a pattern. If pivots sometimes use tabs, and sometimes arrows, then people will not understand what to do. Given that we don't want to add to the number of tabs, especially in site which may have lots of pivots, using arrows makes the most sense.

**Comment by kwilkins** [comment]
> I am trying to repro the problem on CodePen with minimal example - it seems to be a bit more complicated than just a Dialog and a Pivot. Will get back to you once I have singled out the issue.

I'm seeing a similar interaction too. Currently we can reproduce this issue in our app but trying to build an isolated CodePen to share it works as expected. Will continue to try but thought I'd second seeing the issue with tabbing on Dialog+Pivot

**Comment by jdhuntington** [comment]
Opened discrete issues for each bug; closing this as a duplicate.
